user_docs/en/userGuide.t2t

```diff
@@ -1061,7 +1061,7 @@ The available logging levels are:
  - If you are concerned about privacy, do not set the logging level to this option.

```

Comment:
Why are you removing these?

Reply:
What do you exactly mean? Actually they are not removed. I just added a dash on the last line after the list. I saw that this seems to be used most of the times in the userguide, probably as the end of the list.